auth/user_mgt.go

```diff
@@ -56,6 +56,37 @@ type UserInfo struct {
 	UID        string `json:"rawId,omitempty"`
 }
 
+// MultiFactorInfoResponse describes the `mfaInfo` of the user record API response
+type MultiFactorInfoResponse struct {
+	MFAEnrollmentID string `json:"mfaEnrollmentId"`
```

Comment:
Suggest also adding `omitempty` to everything (just for consistency with the rest of this file).